practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/converter/ModelPractitionerConverter.java

```diff
@@ -34,8 +36,11 @@
 
         if (StringUtils.hasText(input.getUserAccountId())) {
             final Identifier identifier = new Identifier();
-            identifier.setValue(input.getUserAccountId());
-            fhirPractitioner.setIdentifier(List.of(identifier));
+            identifier.setUse(IdentifierUse.OFFICIAL);
+            identifier.setId("IDP_IDENTITY");
+            identifier.setValueElement(new StringType(input.getUserAccountId()));
+            identifier.setValue(input.getUserAccountId());            
```

Comment:
What is the difference between these two?

Reply:
None at all, removed!

Reply:
setValue null checks before creating seems to be the only difference - we don;t need both as they do the same job, will remove one, thought I had already